proxmoxer/backends/base_ssh.py

```diff
@@ -81,8 +81,8 @@ class JsonSimpleSerializer(object):
     def loads(self, response):
         try:
             return json.loads(response.content)
-        except ValueError:
-            return response.content
+        except (UnicodeDecodeError, ValueError):
```

Comment:
Why not `JSONDecodeError` ?

Reply:
JSONDecodeError was introduced in the json package used by python3.5. Since I'd like to keep backward compatibility if possible, It is just using ValueError which is the superclass of JSONDecodeError when present and is what was used before JSONDecodeError.